app/controllers/organizations/tasks_controller.rb

```diff
@@ -1,28 +1,39 @@
 class Organizations::TasksController < Organizations::BaseController
-  skip_verify_authorized only: %i[cancel]
-
-  before_action :set_pet, only: %i[new create edit update destroy cancel]
-  before_action :set_task, only: %i[edit update destroy]
+  before_action :set_pet, only: %i[new show index create edit update destroy]
+  before_action :set_task, only: %i[show edit update destroy]
 
   def new
     authorize! Task, context: {pet: @pet}
 
     @task = @pet.tasks.build
   end
 
+  def index
+    authorize! Pet, context: {organization: Current.organization}
```

Comment:
This was close to correct, but with this setup it uses the PetPolicy, not the TaskPolicy. Update this to:
```ruby
authorize! Task, context: {pet: @pet}
```
Action Policy infers the policy based on the class of the first parameter object given to `authorize!`, so now this will grab the correct policy.

The policy was not setup for `index` though since it did not exist so please also update this:
```ruby
# app/policies/organizations/task_policy.rb
  alias_rule :create?, :new?, :index?, to: :manage? # Add `:index?` to line 8 like shown
```
This small change will use the default `manage?` check for index?, which will work fine for this.